apps/frontend/src/main.tsx

```diff
@@ -0,0 +1,18 @@
+import { StrictMode } from 'react';
+import * as ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+
+import App from './app/app';
+import { store } from './app/store';
+
+ReactDOM.render(
+  <StrictMode>
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
```

Comment:
TICK: usually (not always but in most cases) App.tsx is the root file that already includes all the providers, and Router.tsx is a separate one.

Reply:
I agree about router but I'm not sure about provider. What if we will want to use our store in app component?